internal/pkg/cli/interfaces.go

```diff
@@ -226,7 +226,7 @@ type wsPipelineReader interface {
 	wsPipelineManifestReader
 }
 
-type wsProjectManager interface {
+type wsAppManager interface {
 	Create(projectName string) error
```

Comment:
Maybe it's time to fix the param name for this interface?

Reply:
Done!